### ISSUE TITLE
[0.9.1.2] Can't see election votes

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoIssues#19203]
I wanted to go back to how people voted in an old election to create a bug about it, but checking the details on http://gs1.play.eco:3011/elections.html?election=27 I noticed the old votes were blank:

![image](https://user-images.githubusercontent.com/964559/98461714-5c5cda00-2163-11eb-8bcf-5c3a398ac522.png)

I remember being able to see those votes. I checked with some currently running elections and got the same problem:

![image](https://user-images.githubusercontent.com/964559/98461754-afcf2800-2163-11eb-97bd-aec5687dc23e.png)

And the election votes are not anonymous either:

![image](https://user-images.githubusercontent.com/964559/98461770-d1301400-2163-11eb-8ca4-9e6c82b6f9db.png)

**Comment by ThePiachu** [comment]
Hmm, you can see who voted for whom, but only on top of the election page, and it's really hard to parse:

![image](https://user-images.githubusercontent.com/964559/98461968-80211f80-2165-11eb-8fd1-9cc82f9c3fd0.png)

**Comment by denysaw** [comment]
Done (PR)
![image](https://user-images.githubusercontent.com/5720101/104668599-b4bad580-56d8-11eb-96ad-363460d062e5.png)

**Comment by SlayksWood** [comment]
Still have an issue for me. I don't see it at all:
![image](https://user-images.githubusercontent.com/45708377/104918810-59256c00-59a6-11eb-9b17-9090018aceb1.png)
Save files with elections:
https://drive.google.com/file/d/17018DA4bAZCO_2GVFPWCcO-B_kdf3Q49/view?usp=sharing

**Comment by denysaw** [comment]
Fixed in PR above. @SlayksWood thanx for game save, helped a lot. It would be cool to put these elections to a `/generatetestdata` endpoint ;)

**Comment by SlayksWood** [comment]
fixed for Referendums:
![image](https://user-images.githubusercontent.com/45708377/105148649-aecb5700-5b13-11eb-8e30-46126c971e02.png) 
But I guess it should look anothers for Elections of Officials:
![image](https://user-images.githubusercontent.com/45708377/105149661-d5d65880-5b14-11eb-90b4-9d82070727ef.png)
Closed this one, I will create a new issue for that after discussing it with @denysaw
